prime-dotnet-webapi/Services/CommunitySiteService.cs

```diff
@@ -99,6 +99,7 @@ public async Task UpdateSiteAsync(int siteId, CommunitySiteUpdateModel updatedSi
             UpdateContacts(currentSite, updatedSite);
             UpdateBusinessHours(currentSite, updatedSite);
             UpdateRemoteUsers(currentSite, updatedSite.RemoteUsers);
+            await UpdateIndividualDeviceProviders(siteId, updatedSite);
```

Comment:
Might as well pass in `updatedSite.IndividualDeviceProviders` rather than the whole update Model.